src/main/sensors/compass.h

```diff
@@ -48,7 +49,8 @@ extern mag_t mag;
 typedef struct compassConfig_s {
     int16_t mag_declination;                // Get your magnetic decliniation from here : http://magnetic-declination.com/
                                             // For example, -6deg 37min, = -637 Japan, format is [sign]dddmm (degreesminutes) default is zero.
-    sensor_align_e mag_align;               // mag alignment
+    sensor_align_e mag_alignment;               // mag alignment
+    sensorAlignment_t mag_customAlignment;
```

Comment:
This has to go at the end, or the config will be corrupted.

Reply:
done